int32 human_id
int32 is_operator
int32 left_hg
int32 right_hg

PointCoors[] body_points
